doc/source/exclude_patterns.md

```diff
@@ -0,0 +1,93 @@
+# Exclude Pattern Settings 
+- Optional argument for [in-toto-run.](https://in-toto.readthedocs.io/en/latest/command-line-tools/in-toto-run.html) Path patterns to match paths that should not be recorded as 'materials' or 'products'.  Passed patterns override patterns defined in environment variables or config files (see ```ARTIFACT_EXCLUDE_PATTERNS``` documentation for additional info) 
```

Comment:
Line length alert!

Let's keep them <= 80. Also note that you don't need to add a space at the end of a line or beginning of the next line.

Reply:
Fixed it up a bit, let me know if the length works!